src/test/java/bio/terra/app/controller/DatasetsApiControllerTest.java

```diff
@@ -378,6 +382,26 @@ void testCreateSnapshotRequest() throws Exception {
     verifyAuthorizationCall(IamAction.VIEW_SNAPSHOT_BUILDER_SETTINGS);
   }
 
+  @Test
+  void testEnumerateSnapshotRequests() throws Exception {
+    mockValidators();
+    EnumerateSnapshotAccessRequestItem responseItem =
+        SnapshotBuilderTestData.createEnumerateSnapshotAccessRequestModelItem();
+    EnumerateSnapshotAccessRequest response = new EnumerateSnapshotAccessRequest();
```

Comment:
It's up to you but I think `var` makes this code more readable.
```suggestion
    var responseItem = SnapshotBuilderTestData.createEnumerateSnapshotAccessRequestModelItem();
    var response = new EnumerateSnapshotAccessRequest();
```